docs/quick-start.md

```diff
@@ -0,0 +1,189 @@
+# Getting started with reignite
+
+## Set up networking
+
+```
+# Default network device
+NET_DEVICE=$(ip route show | awk '/default/ {print $5}')
+
+# Create a tap device
+sudo ip tuntap add tap0 mode tap
+sudo ip addr add 172.100.0.1/24 dev tap0
+sudo ip link set tap0 up
+
+# Forward tap to default device
+sudo iptables -A FORWARD -i tap0 -o $NET_DEVICE -j ACCEPT
+
+# MAC address for tap0
+# Can be useful later
+export TAP0_MAC="$(cat /sys/class/net/tap0/address)"
+```
+
+### Create thinpool
+
+Easy quick-start option is to run this script as root. I know,
+it's not recommended in general, and I'm happy you think it's not a good
+way to do things, so there are comments for commands.
+
+```bash
```

Comment:
I use a slightly different script:

```bash
#!/bin/bash
set -ex

DATA_DIR=/var/lib/containerd/devmapper
POOL_NAME=devpool

mkdir -p ${DATA_DIR}

# Create data file
sudo touch "${DATA_DIR}/data"
sudo truncate -s 100G "${DATA_DIR}/data"

# Create metadata file
sudo touch "${DATA_DIR}/meta"
sudo truncate -s 10G "${DATA_DIR}/meta"

# Allocate loop devices
DATA_DEV=$(sudo losetup --find --show "${DATA_DIR}/data")
META_DEV=$(sudo losetup --find --show "${DATA_DIR}/meta")

# Define thin-pool parameters.
# See https://www.kernel.org/doc/Documentation/device-mapper/thin-provisioning.txt for details.
SECTOR_SIZE=512
DATA_SIZE="$(sudo blockdev --getsize64 -q ${DATA_DEV})"
LENGTH_IN_SECTORS=$(bc <<< "${DATA_SIZE}/${SECTOR_SIZE}")
DATA_BLOCK_SIZE=128
LOW_WATER_MARK=32768

# Create a thin-pool device
sudo dmsetup create "${POOL_NAME}" \
    --table "0 ${LENGTH_IN_SECTORS} thin-pool ${META_DEV} ${DATA_DEV} ${DATA_BLOCK_SIZE} ${LOW_WATER_MARK}"

cat << EOF
#
# Add this to your config.toml configuration file and restart containerd daemon
#
[plugins]
  [plugins.devmapper]
    pool_name = "${POOL_NAME}"
    root_path = "${DATA_DIR}"
    base_image_size = "10GB"
    discard_blocks = true
EOF
```

Reply:
It seem the same, but without extra checks (if it's already there, if the file or the loop device is already associated).

Reply:
Added the extra echo note at the end of the script and change the metadata volume to 10G from 2G.